### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by jheart-vic**
I created an index.html and a style.css file with the HTML file styled

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1{
+    color: aqua;
+}
```
Comment (prolajumokeoni) [inline style.css:3]: - [ ] Kindly link your style.css page to the index.html page so your font color is updated 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used



## Required Changes ♻️

- [ ] Kindly fix Stylelint errors, your project should pass all linter checks 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) and [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) to your root project so you can set up linters locally, also don't forget to add node_modules to .gitignore so you don't add it to git 🤝 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jheart-vic** [comment]
Please is the .gitignore supposed to be a folder

File: style.css (hunk 1 above)
Comment (jheart-vic) [reply style.css:3]: > * [ ]  Kindly link your style.css page to the index.html page so your font color is updated 👍🏿

Thanks
I will be careful next time

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
[the remark this answers is not in the document]
Comment (jheart-vic) [reply test.md:1]: please @prolajumokeoni is the .gitignore suppose to be a folder

[the remark this answers is not in the document]
Comment (jheart-vic) [reply test.md:1]: Please do I have to redo it @prolajumokeoni

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @jheart-vic ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 


parks-and-rec-great-job-gif

## Optional suggestions

- [ ] I strongly suggest to update your readme following best practices, currently some sections are missing. You can use [this template](https://github.com/microverseinc/readme-template) provided by microvesre. 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

***
Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**